smarts-imitation/smarts_imitation/utils/common.py

```diff
@@ -226,11 +226,13 @@ def cal_neighbor_with_radius_ego_coordinate(env_obs: Observation, **kwargs):
         )
         for i, v in surrounding_vehicles.items():
             if v[0] is None:
-                dist = 15.0
+                # dist = 15.0
+                dist = -5.0
                 heading_vec = [0.0, 0.0]
                 speed = -1.0
             else:
-                dist = min(v[1], 15.0)
+                # dist = min(v[1], 15.0)
+                dist = v[1]
```

Comment:
What is the change here for?

Reply:
It performs better in my experiments (remove the max distance clipping and zero-padding the vehicle-no-found cases)